frontend/pages/hosts/details/cards/MunkiIssues/MunkiIssues.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useMemo } from "react";
+
+import { IMunkiIssue } from "interfaces/host";
+import TableContainer from "components/TableContainer";
+
+import EmptyState from "../EmptyState";
+
+import { generateMunkiIssuesTableHeaders } from "./MunkiIssuesTableConfig";
+
+const baseClass = "munki-issues";
+
+interface IMunkiIssuesTableProps {
+  isLoading: boolean;
+  munkiIssues?: IMunkiIssue[];
+  deviceType?: string;
+}
+
+const MunkiIssuesTable = ({
+  isLoading,
+  munkiIssues,
+  deviceType,
+}: IMunkiIssuesTableProps): JSX.Element => {
+  const tableMunkiIssues = useMemo(() => munkiIssues, [munkiIssues]);
+  const tableHeaders = generateMunkiIssuesTableHeaders();
+
+  const EmptyMunkiIssues = () => {
```

Comment:
This can be moved to above line 18 so that the empty component isn't being redefined each render.